source/api_reference/index.html.md.erb

```diff
@@ -68,7 +72,8 @@ The response will include a ``status`` value as described in the table below, an
 
 ## HTTP status codes
 
-You will encounter typical HTTP success and error response codes when using the GOV.UK Pay API. Generally any:
+You will encounter typical HTTP success and error response codes when using
```

Comment:
If this is just general discussion about generic HTTP response codes can we link to w3 or somewhere instead?

Reply:
HTTP (and hence its status codes) are an IETF thing, not a W3C thing.

I can’t find any authoritative document that outlines HTTP status codes without detailing each one, which would include ones we don’t use (and if often biased towards using HTTP for web browsing).